cmd/xl-v1-object.go

```diff
@@ -249,7 +249,11 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 	chunkSize := getChunkSize(xlMeta.Erasure.BlockSize, xlMeta.Erasure.DataBlocks)
 	pool := bpool.NewBytePool(chunkSize, len(onlineDisks))
 
-	// Read from all parts.
+	storage, err := NewErasureStorage(onlineDisks, xlMeta.Erasure.DataBlocks, xlMeta.Erasure.ParityBlocks, xl.readQuorum, xl.writeQuorum)
```

Comment:
We should not use xl.readQuorum here but instead use it from xlMeta (as they can be different!)

Or rather use xlMeta.Erasure.DataBlocks as the read quorum for Gets